script.js

```diff
@@ -2,27 +2,60 @@ var values  = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A"];
 var suits   = ["Clubs", "Diamonds", "Hearts", "Spades"];
 var game = {
   deck: [],
-  players: [],
-  hands: [],
+  players: [2],
+  hands: [4]
+var instructor = {
+  name: "Players"
```

Comment:
It looks like you're trying to write an object here. Objects are key/value pairs. There always has to be a comma between each value:

```js
var hobbit = {
  name: "Bilbo",
  jewelry: "Ring"
}
```